tests/validation/tests/v3_api/test_bkp_restore.py

```diff
@@ -65,6 +79,50 @@ def test_bkp_restore():
     assert len(workload_list) == 0
 
 
+def test_s3_snapshot_delete(node_template_ec2_iam):
```

Comment:
Can we rename the method to include "create" scenario also, something like test_s3_snapshot_create_delete ?

Reply:
good catch. Renamed